app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-confirm-step.tsx

```diff
@@ -0,0 +1,51 @@
+import { Alert } from '@18f/identity-components';
+import { FormStepsContext, FormStepsContinueButton } from '@18f/identity-form-steps';
+import { t } from '@18f/identity-i18n';
+import type { FormStepComponentProps } from '@18f/identity-form-steps';
+import { Modal } from '@18f/identity-modal';
+import PersonalKeyStep from '../personal-key/personal-key-step';
+import PersonalKeyInput from './personal-key-input';
+import type { VerifyFlowValues } from '../..';
+
+interface PersonalKeyConfirmStepProps extends FormStepComponentProps<VerifyFlowValues> {}
+
+function PersonalKeyConfirmStep(stepProps: PersonalKeyConfirmStepProps) {
+  const { registerField, errors } = stepProps;
+
+  return (
+    <>
+      <FormStepsContext.Provider value={{ isLastStep: false, onPageTransition() {} }}>
```

Comment:
Since we're reusing the PersonalKeyStep implementation, the context here is necessary to trick the button in the background to show as "Continue", rather than "Submit", since the default form context would think it's now at the last step (confirmation step).

![Screen Shot 2022-04-15 at 10 57 01 AM](https://user-images.githubusercontent.com/1779930/163586218-fc2c29b6-a869-4c86-b695-9dffbba7a605.png)